### DISCUSSION TITLE
Limiting options shown in the dropdown

### DISCUSSION THREAD

**Discussion by AaronPlave** [janosh/svelte-multiselect#241] [Q&A] [answered]
Is there any way to limit the number of options rendered in the dropdown without interfering with the filtering capabilities? I'm looking to use this component as more of a tags input with autocomplete as opposed to a true multiselect and anticipate that I may need to provide thousands of options which would be too much to show in the dropdown.

**Comment by janosh** [comment, marked as answer]
I just opened #243 that adds a `maxOptions` prop. You can see it in action on the landing page in the [`countries` example](https://multiselect.janosh.dev/#--examples). Does this serve your purpose?

**Reply by AaronPlave** [reply]
Perfect, it sure does! Thank you for the speedy response!

**Reply by janosh** [reply]
Great! Will be part of the next release. No firm date yet.
